includes/definitions/discovery/ciena-saos.yaml

```diff
@@ -0,0 +1,4 @@
+modules:
+    os:
+        sysDescr_regex: '/(?<hardware>.*)SW:(?<version>\S+)/'
+        serial: SNMPv2-SMI::mib-2.47.1.1.1.1.11.537001984
```

Comment:
This OID is in fact `ENTITY-MIB::entPhysicalSerialNum`. This is not a problem, except that the index 537001984 does not look "standard". It may only work on your chassis.
Can you find the same serial number in the vendor MIB, in a more predictable OID ? (And if yes, please add that MIB, and use it instead.)

Reply:
Negative, this was the only location where I could find the chassis serial and the serials for all the other cards.